sswf-core/src/it/scala/com/bazaarvoice/sswf/ListOpenExecutionTest.scala

```diff
@@ -77,18 +78,21 @@ class ListOpenExecutionTest extends FlatSpec {
     assert(currentOpenExecutions.size() == 0)
 
     val closedDate = new Date()
-    val closedExecutions = manager.listClosedExecutions(startTime, closedDate, workflowId)
+    // TODO possible bug in AWS SDK used with this version, removing workflowId param returns the closed workflow here
+    val closedExecutions = manager.listClosedExecutions(startTime, closedDate)
     assert(closedExecutions.size() == 1)
 
     assert(closedExecutions.get(0).getExecutionStatus == "CLOSED")
     assert(closedExecutions.get(0).getExecution.getWorkflowId == workflowId)
     assert(closedExecutions.get(0).getStartTimestamp.after(startTime))
-    assert(closedExecutions.get(0).getCloseTimestamp.before(closedDate))
+    // TODO possible bug in AWS SDK used with this version, timestamp in returned object fails this assertion
```

Comment:
the only thing we could do is try another version of the AWS SDK in Doula and in this library.

Reply:
Realized this morning that this might be another timing bug and that what the test might need is a second thread sleep *before* querying for the list of closed executions, i.e. allow the SSWF to get the current workflow into the closed state before querying it. I did this and sure enough it worked, so I removed the TODO statements and put the code back the way it was.